hail/www/references.md

```diff
@@ -1,10 +1,20 @@
 # Hail-Powered Science
 
-The following is an incomplete list of scientific work enabled by Hail. We welcome you to add additional examples by [editing this page directly](https://github.com/hail-is/hail/edit/master/hail/www/references.md), after which we will review the pull request to confirm the addition is valid. Please adhere to the existing formatting conventions.
+The following is an incomplete list of scientific work enabled by Hail.
 
-In addition to software development, the Hail team engages in theoretical, algorithmic, and empirical research inspired by scientific collaboration. Examples include [Loss landscapes of regularized linear autoencoders](https://github.com/danielkunin/Regularized-Linear-Autoencoders), [Secure multi-party linear regression at plaintext speed](https://github.com/jbloom22/DASH), and [A synthetic-diploid benchmark for accurate variant-calling evaluation](https://www.nature.com/articles/s41592-018-0054-7).
+If you use Hail for published work, please cite the software.
 
-*Last updated on February 3rd, 2020 at 11:00 AM EST*
+```
+Hail Team. Hail 0.2. https://github.com/hail-is/hail
+```
+
+Otherwise, we welcome you to add additional examples by [editing this page directly](https://github.com/hail-is/hail/edit/master/hail/www/references.md), after which we will review the pull request to confirm the addition is valid. Please adhere to the existing formatting conventions.
+
+*Last updated on March 30th, 2020 at 11:00 AM EST*
```

Comment:
can we delete the time? No reason to have hour/minute here 😄

Reply:
done :)